cloud/pkg/synccontroller/objectsync.go

```diff
@@ -71,6 +93,12 @@ func (sctl *SyncController) manageObject(sync *v1alpha1.ObjectSync) {
 
 func sendEvents(err error, nodeName string, sync *v1alpha1.ObjectSync, resourceType string,
 	objectResourceVersion string, obj interface{}) {
+	// some workaround to keep the CRD resource type consistent with the edge side
+	// TODO: investigate to have a more systematic way to handle.
```

Comment:
let's create an issue for todo. TODOs in code tend to get ignored and lost.

Reply:
tracking issue : https://github.com/CentaurusInfra/fornax/issues/27